Front-End/Javascript-Intermidiate/CRUD.js

```diff
@@ -0,0 +1,11 @@
+//This is creating the element in js
+let element = document.createElement('p');
+element.className = 'custom-made';
+element.id = 'custom-created';
+element.innerText = 'This is not the random text. This is create by kaiwalya';
+
```

Comment:
This finally creates a `<p  class = "custom-made" id = "custom-created">This is not the random text. This is create by kaiwalya</p>`